Rosyk_Task4.py

```diff
@@ -0,0 +1,25 @@
+# Multiparadigm programming languages, Task 2
+# Rosyk Nikita, №15
+
+print('Multiparadigm programming languages, Task 2')
+print('Nikita Rosyk gr. IKM-221k, №15')
+
+def is_prime(num):
+    if num <= 1:
+        return False
+    for i in range(2, int(num ** 0.5) + 1):
```

Comment:
Looks so mysterious.
I'd like to discuss the code in future.
please recall me during the class or just ping when it would be convenient for you

Reply:
cannot remember what I did mean. sorry for the stupid comment